app/bundles/LeadBundle/Model/ListModel.php

```diff
@@ -11,6 +11,7 @@
 
 use Mautic\CoreBundle\Helper\CoreParametersHelper;
 use Mautic\CoreBundle\Helper\DateTimeHelper;
+use DateInterval;
```

Comment:
This use statement is not needed when you use `new \DateInterval('PT23H59M59S')` (with the `\`) later in the code.